src/components/Visualizations/inflictorWithValue.jsx

```diff
@@ -40,10 +40,9 @@ const tooltipContainer = thing => (
       <div
         dangerouslySetInnerHTML={{
           __html: thing.cmb
-            .replace('src="/apps/dota2/images/tooltips/mana.png"', `src="${API_HOST}/apps/dota2/images/tooltips/mana.png"`)
-            .replace('src="/apps/dota2/images/tooltips/cooldown.png"', `src="${API_HOST}/apps/dota2/images/tooltips/cooldown.png"`)
-            .replace('http://cdn.dota2.com/apps/dota2/images/tooltips/mana.png', `${API_HOST}/apps/dota2/images/tooltips/mana.png`)
-            .replace('http://cdn.dota2.com/apps/dota2/images/tooltips/cooldown.png', `${API_HOST}/apps/dota2/images/tooltips/cooldown.png`),
+            .replace(/http:\/\/cdn\.dota2\.com/g, '')
```

Comment:
see my updated comment (second arg should be API_HOST).

You also don't need the next two lines, since cdn.dota2.com is now left intact.